.travis.yml

```diff
@@ -20,6 +21,7 @@ env:
   - MONGO_DB_NAME="ppp-board-test"
   - TF_ENV_PREFIX="prod"
   - TF_VERSION=0.14.7
+  - LOG_LEVEL="ERROR"
```

Comment:
Error is a reasonable setting for production, but unless you know your logs are too noisy I'd always be inclined to include warnings too